test/integration_test.py

```diff
@@ -784,27 +784,40 @@ def _source_ids_from_hits(hits: JSONs) -> Set[str]:
                    args={
                        'catalog': catalog,
                        'filters': json.dumps(source_filter)
-                   }).url
-        response = self._get_url_json(url)
+                   })
+        response = self._get_url_json(url.url)
         self.assertEqual(response['hits'], [])
         self.assertEqual(response['pagination']['total'], 0)
 
         with self._http_context(self._service_account_authorized_http):
-            response = self._get_url_json(url)
+            response = self._get_url_json(url.url)
         indexed_source_ids = _source_ids_from_hits(response['hits'])
         self.assertEqual(indexed_source_ids, managed_access_source_ids)
-        managed_access_files = (
-            file['url']
-            for bundle in response['hits']
-            for file in bundle['files']
-        )
-        try:
-            file_url = first(managed_access_files)
-        except ValueError:
-            pass
-        else:
+
+        with self._http_context(self._service_account_authorized_http):
+            response = self._get_url_json(url.set(path='/index/files').url)
+        managed_access_file_count = response['pagination']['total']
+        if managed_access_file_count > 0:
+            file_url = one(first(response['hits'])['files'])['url']
             response = self._http.request('GET', file_url)
             self.assertEqual(response.status, 404)
+        summary_url = furl(config.service_endpoint(),
+                           path='/index/summary',
+                           args={'catalog': catalog}).url
+
+        def _get_summary_file_count() -> int:
+            response = self._get_url_json(summary_url)
+            # FIXME: https://github.com/DataBiosphere/azul/issues/3318
```

Comment:
```suggestion
            # FIXME: fileCount doesn't equal the sum of fileTypeSummaries[].count
                     https://github.com/DataBiosphere/azul/issues/3318
```

The FIXME should be placed where `fileCount` and/or `fileTypeSummaries[].count` are calculated.